### PR TITLE
Docs/refactor project name

### PR DESCRIPTION
**Pull request by GH-Syn**
## Changes
 - cleaned up a few imports 
 - Minor typing
 - small readability changes

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -54,11 +47,12 @@ def accept_user_input():
             # Instantiate IPReputationChecker class
             breach_checker = EmailBreachChecker(args.ip)
```
Comment (initd1) [inline main.py:48]: this needs to be args.email

**Review by initd1 — CHANGES_REQUESTED** [review]
the email argument in mail.py needs to be changed from 
`breach_checker = EmailBreachChecker(args.ip)` to
`breach_checker = EmailBreachChecker(args.email)`

File: main.py (hunk 1 above)
Comment (GH-Syn) [reply main.py]: Done :+1:

File: README.md [hunk 2]
```diff
@@ -12,24 +12,34 @@
...
 HIBP_APIKey = 1234567890
 ```
+
 3. Add API keys for Virus Total and Haveibeenpwned
 
 ## Execution
 1. HackAlert Usage:
-```
+```bash
 python main.py -h
```
Comment (initd1) [inline README.md:43]: please update usage to `python hackert.py -h`

**Review by initd1 — CHANGES_REQUESTED** [review]

**Review by initd1 — APPROVED** [review]

**Review by initd1 — APPROVED** [review]

**Pull request merged**